src/main/java/namor/template/stack/ArrayStack.java

```diff
@@ -0,0 +1,71 @@
+package namor.template.stack;
+
+import java.util.Arrays;
+
+// TODO: 20.03.17 [GUSb] A test class to check everything works, should be deleted with first finished task
+public class ArrayStack {
+
+    private int top;
+    private int size;
+    private int[] stack;
+
+    public ArrayStack(int size)
+    {
```

Comment:
Can i use here IllegalArgumentException in case if someone will try to inject value < 1?